Live_data_update.sql

```diff
@@ -19,3 +21,85 @@ update cqc."Establishment" set "IsRegulated"=True,"LocationID"='1-5400068426' wh
 select * from cqc."Establishment" where "EstablishmentID"=337 ;
 
 
+-- Date 16 May 2019
+
+-- https://trello.com/c/Ypwr3hZ4/55-remove-parent-cohort-2-users-from-asc-wds
+
+--Deletion data view before we delete the Establishment and related records
+
+-- Selection based on Tribal ID [196840,170258]
+
+
+select "TribalID"   from cqc."Establishment" where "NmdsID" IN ('E302906','G260465');
+
+SELECT * FROM cqc."Worker" where "TribalID" in (select "TribalID" from cqc."Establishment" where "NmdsID" IN ('E302906','G260465')) ;
```

Comment:
Why are you returning the whole of the records? Isn't it enough to return the count of records?

Reply:
Agree, I will edit it to make it count instead of returning the whole records.

Thanks
Shakir

Reply:
Hi Warren 

I have changed the records to counting the records.

Thanks
Shakir